### DISCUSSION TITLE
Delete email from List

### DISCUSSION THREAD

**Discussion by chaitanyaj2222** [RealDevSquad/Centauri-React-Native-Cohort#19] [Show and tell]
### Todo
* Swipe an item to delete it from the list
[About the assignment](https://github.com/Real-Dev-Squad/Centauri-React-Native-Cohort/blob/main/react-native/action-items/2022-01/2022-01-15.md)

**Comment by chaitanyaj2222** [comment]
https://user-images.githubusercontent.com/56331036/150686109-f4db0e8d-1feb-4574-b761-a2589f927dea.mp4

**Comment by nandanpandey43** [comment]
# done with swipe to delete
github link :- https://github.com/nandanpandey43/RDS-React-Native-Cohort-Apps/tree/master/assignmentProject

https://user-images.githubusercontent.com/38531527/150978446-ef44a37c-62b4-4e89-9840-7554fe32766b.mp4

**Comment by Sumanta1998** [comment]
Done with Swipe to delete
https://user-images.githubusercontent.com/44142633/151214206-89fe6860-9ba7-4918-a4b6-75b4b0661ac7.mp4

**Comment by mohankumar80** [comment]
https://user-images.githubusercontent.com/50794740/151552139-4a3e7dc3-61f9-409e-a867-2283cfc274d8.mp4

**Comment by ParakhBansal** [comment]
https://user-images.githubusercontent.com/65717231/151776461-5cc03170-40ad-4e16-bf92-1f439098f86d.mp4

**Comment by Arnab-31** [comment]
https://user-images.githubusercontent.com/57807670/151880999-13881070-539e-4741-8d15-329d6f96aaf8.mp4

**Comment by prachic17** [comment]
https://user-images.githubusercontent.com/61531836/152336305-34b392b3-400e-4281-9145-58338f0944c5.mp4

**Comment by jatin33** [comment]
Done with swipe to delete and undo the deleted item as well
https://youtu.be/GapYXYJhaoM
Repo: https://github.com/jatin33/RDS-react-native-swipeable-list-task
